docs/iris/example_code/General/coriolis_plot.py

```diff
@@ -0,0 +1,99 @@
+"""
+Coriolis parameters over the UK
+===============================
+
+This example shows the coriolis parameters for the area encompassing the UK.
+
+It creates and plots a map showing the coriolis parameters, overlaid on top of
+the UK coastline. The code also creates a custom cube to store the data in and
+to plot it from.
+
+"""
+
+import cartopy.crs as ccrs
+
+import iris
+
+from iris.coord_systems import GeogCS
+
+import iris.quickplot as qplt
+
+import matplotlib.pyplot as plt
+
+import numpy as np
+
+
+def calculate_coriolis_parameter(latitude):
+    """
+    This function computes the coriolis parameters for the points in an array.
+
+    It takes a 2D latitude array as its input.
+
+    It returns a 2D array for coriolis parameters.
+    """
+    omega = 7.29E-5
+    return 2. * omega * np.sin(latitude)
+
+
+def main():
+    # Start with arrays for latitudes and longitudes, with a given resolution.
+    resolution = 50
+    latitudes = np.linspace(49.0, 60.0, resolution)
+    longitudes = np.linspace(-11.0, 3.0, resolution)
+    lat_calc = np.deg2rad(latitudes)
+    lon_calc = np.deg2rad(longitudes)
+    lat1, long1 = np.meshgrid(lat_calc, lon_calc)
+
+    # Now to call the coriolis function
+    data = calculate_coriolis_parameter(lat1)
+
+    # We now need to define a coordinate system for the plot.
+    # Here we'll use GeogCS; 6371229 is the radius of the Earth in metres.
+    cs = GeogCS(6371229)
+
+    # The Iris coords module turns the latitudes list into a coordinate array.
+    # Coords then applies an appropriate standard name and unit to it.
+    lat_coord = iris.coords.DimCoord(latitudes,
+                                     standard_name='latitude',
+                                     units='degrees',
+                                     coord_system=cs)
+
+    # The above process is repeated for the longitude coordinates.
+    long_coord = iris.coords.DimCoord(longitudes,
+                                      standard_name='longitude',
+                                      units='degrees',
+                                      coord_system=cs)
+
+    # The next step is to add vertical structure to the cube.
+    # This cube has only one level, but a cube with multiple levels
+    # can be made by feeding in a list rather than a single value.
+    height_coord = iris.coords.DimCoord(100.,
+                                        bounds=[90., 110.],
+                                        standard_name=None,
+                                        units='m',
+                                        long_name='level_height',
+                                        attributes={'positive': 'up'})
+
+    # Now we input our data array into the cube.
+    new_cube = iris.cube.Cube(data,
+                              standard_name='coriolis_parameter',
+                              units='s-1',
+                              dim_coords_and_dims=[(long_coord, 0),
```

Comment:
As I've suggested elsewhere, I'd prefer the order [latitude, longitude]. If you make the suggested change to the `meshgrid` call this will be necessary.